seed_stage_based_messaging/settings.py

```diff
@@ -234,37 +219,26 @@
 
 METRICS_REALTIME = [
     'subscriptions.created.sum',
-    'subscriptions.send_next_message_errored.sum'
+    'subscriptions.send_next_message_errored.sum',
+    'sbm.send_next_message.connection_error.sum',
```

Comment:
Not sure I understand why these were renamed?

Reply:
They weren't renamed, they were just wrong in the first place. The task fires them as `sbm...`, but in the metrics list they appeared as `subscriptions...`, so if we tried to use these metrics, they wouldn't have worked.